OSDOCS-3531--OSDK-1738: File-based catalog updates for `run bundle` (#47740)
openshift/openshift-docs: modules/osdk-deploy-olm.adoc

    --olm-namespace=openshift-operator-lifecycle-manager
----

> . Run the Operator on your cluster by using the OLM integration in Operator SDK to create a valid file-based catalog installs the Operator bundle on the cluster:

Reviewer: ```suggestion
. Run the Operator on your cluster by using Operator SDK that uses OLM to create a valid file-based catalog thereby installing the Operator bundle on the cluster:
```

    --olm-namespace=openshift-operator-lifecycle-manager
----

. Enter the following command to run the Operator on the cluster: 
+
[source,terminal]
----
$ operator-sdk run bundle \//<1>

>     [-n <namespace>] \//<2>

Reviewer 2: This is more of an observation, but I've never seen -n <namespace> in brackets, and I'm not sure if it would work with brackets (in the past I've only used it as, for example, "$ operator-sdk run bundle -n <namespace> <registry>/<user>/<bundle_image_name>:<tag>".

I know it passed QE. Maybe it does work with brackets?

Author: I added an `Optional:` at the start of the callout and removed the brackets to be more comformant with our practices.

    <registry>/<user>/<bundle_image_name>:<tag> <3>
----

> <1> The run bundle command creates a valid file-based catalog and installs the Operator bundle on your cluster using OLM.

Reviewer 2: Would run bundle be in back ticks?

----
$ operator-sdk run bundle \
    [-n <namespace>] \//<1>
    <registry>/<user>/<bundle_image_name>:<tag> <2>
----
<1> By default, the command installs the Operator in the currently active project in your `~/.kube/config` file. You can add the `-n` flag to set a different namespace scope for the installation.

> <2> If you do not specify an image, the command creates a valid file-based catalog using the default index image of `quay.io/operator-framework/opm:latest`. As a result, the command installs the Operator bundle on cluster. 

Reviewer: ```suggestion
<2> If you do not specify an image, the command creates a valid file-based catalog using the default index image of `quay.io/operator-framework/opm:latest`. As a result, the command installs the Operator bundle on cluster. 

If you do specify an image, the command generates a valid file-based catalog with the bundle image itself and installs the operator bundle on the cluster.
```
